docs/basics.adoc

```diff
@@ -1037,7 +1037,7 @@ Suppose that Hercules battled hundreds of monsters in addition to the three capt
 h = g.V().has('name', 'hercules').next()
 g.V(h).outE('battled').has('time', inside(10, 20)).inV()
 
-Building a vertex-centric index by time speeds up such traversal queries.
+Building a vertex-centric index by time speeds up such traversal queries. Note, this initial index example already exists in the graph of gods as an index named edges. As a result, running the steps below will result in a uniqueness constraint error.
```

Comment:
Looks good, just two small nits
* It looks like graph of gods is referred to fairly consistently as *Graph of the Gods* so it would be nice to match that
* perhaps you could make edges `edges` instead (code styled)